README.md

```diff
@@ -100,14 +100,21 @@ Open the rule in SonarQube / SonarCloud, scroll down and (in case the rule has p
 
 Use SonarLint Connected Mode to connect to SonarQube and SonarCloud.
 
-### standalone nuget / SonarLint
+### SonarLint
 
-You can see how we do it in our own repository for SonarLint Connected mode (and you can reuse the same concept for the nuget package):
+The easiest way is to configure a Quality Profile in SonarCloud.
+
+* Create a dummy repository and analyze it in SonarCloud (it's free for open-source).
+* Configure the Quality Profile for the project you created.

```

Comment:
Let's imagine that reader doesn't know SonarCloud and where to look for the `Quality Profile` thing:
```suggestion
* Configure the Quality Profile in SonarCloud for the project you created.

```